lib/node_modules/@stdlib/math/base/special/pow/src/main.c

```diff
@@ -0,0 +1,725 @@
+*/
+static double polyval_p( const double x ) {
+	return 0.16666666666666602 + (x * (-0.0027777777777015593 + (x * (0.00006613756321437934 + (x * (-0.0000016533902205465252 + (x * 4.1381367970572385e-8)))))));
+}
+
+// END: polyval_p
+
+/* End auto-generated functions. */
+
+/**
+* Evaluates the exponential function when \\( y = \pm \infty\\).
+*
+* @param x    input value
+* @param y    input value
+* @return	  output value
+*
+* @example
+* double out = y_is_infinite( -1.0, Infinity );
+* // returns NaN
+*
+* @example
+* double out = y_is_infinite( 1.5, -Infinity );
+* // returns 0.0
+*/
+double y_is_infinite( const double x, const double y ) {
+    if ( x == -1.0 ) {
+        return 0.0 / 0.0; // NaN
+    }
+    if ( x == 1.0 ) {
+        return 1.0;
+    }
+	// (|x| > 1 && y == NINF) || (|x| < 1 && y === PINF)
+	if ( (stdlib_base_abs( x ) < 1.0) == (y == STDLIB_CONSTANT_FLOAT64_PINF) ) {
+		return 0.0;
+	}
+	// (|x| > 1 && y == PINF) || (|x| < 1 && y == NINF)
+    return STDLIB_CONSTANT_FLOAT64_PINF;
+}
+
+/**
+* Evaluates the exponential function when \\(|y| > 2^64\\).
+*
+* @param x    input value
+* @param y    input value
+* @return	  output value
+*
+* @example
+* double out = y_is_huge( 9.0, 3.6893488147419103e19 );
+* // returns Infinity
+*
+* @example
+* double out = y_is_huge( -3.14, -3.6893488147419103e19 );
+* // returns 0.0
+*/
+double y_is_huge( const double x, const double y ) {
+	uint32_t hx;
+	double ahx;
+	double xc;
+
+	xc = x;
+    stdlib_base_float64_get_high_word( xc, &hx );
+    ahx = ( hx & STDLIB_CONSTANT_FLOAT64_HIGH_WORD_ABS_MASK );
+
+    if ( ahx <= HIGH_MAX_NEAR_UNITY ) {
+        if ( y < 0 ) {
+			// Signal overflow...
+            return HUGE * HUGE;
+        }
+		// Signal underflow...
+        return TINY * TINY;
+    }
+	// `x` has a biased exponent greater than or equal to `0`...
+
+    if ( y > 0 ) {
+		// Signal overflow...
+        return HUGE * HUGE;
+    }
+	// Signal underflow...
+	return TINY * TINY;
+}
+
+/**
+* Evaluates the exponential function when \\(|x| = 0\\).
+*
+* @param x    input value
+* @param y    input value
+* @return	  output value
+*
+* @example
+* double out = x_is_zero( 0.0, 2 );
+* // returns 0.0
+*
+* @example
+* double out = x_is_zero( 0.0, -9 );
+* // returns Infinity
+*/
+double x_is_zero( const double x, const double y ) {
+    if ( y == STDLIB_CONSTANT_FLOAT64_NINF ) {
+        return STDLIB_CONSTANT_FLOAT64_PINF;
+    }
+	if ( y == STDLIB_CONSTANT_FLOAT64_PINF ) {
+		return 0.0;
+	}
+	if ( y > 0.0 ) {
+		if ( stdlib_base_is_odd( y ) ) {
+			return x; // handles +-0
+		}
+		return 0.0;
+	}
+	// y < 0.0
+	if ( stdlib_base_is_odd( y ) ) {
+		return stdlib_base_copysign( STDLIB_CONSTANT_FLOAT64_PINF, x ); // handles +-0
+	}
+	return STDLIB_CONSTANT_FLOAT64_PINF;
+}
+
+/**
+* Computes \\(2^{\mathrm{hp} + \mathrm{lp}\\).
+*
+* @param j    high word of hp + lp
+* @param hp    first power summand
+* @param lp    second power summand
+* @return	  function value
```

Comment:
```suggestion
* @param j    high word of hp + lp
* @param hp    first power summand
* @param lp    second power summand
* @return	  function value
```

Align descriptions. You've also mixed tabs and spaces.